src/main/java/net/coderbot/iris/Iris.java

```diff
@@ -38,7 +38,7 @@
 	public static final String MODID = "iris";
 	public static final Logger logger = LogManager.getLogger(MODID);
 
-	private static final Path shaderpacksDirectory = FabricLoader.getInstance().getGameDir().resolve("shaderpacks");
+	public static final Path shaderpacksDirectory = FabricLoader.getInstance().getGameDir().resolve("shaderpacks");
```

Comment:
should this be in snake case?

Reply:
Correct. Although I do have a question - shouldn't the private static final fields also be in screaming snake case? Or have I misunderstood java conventions this whole time

Reply:
Will mark conversation as resolved